cmd/formatter/colors.go

```diff
@@ -37,7 +37,7 @@ var names = []string{
 
 const (
 	BOLD      = "1"
-	FAINT     = "2"
+	DIM       = "2"
```

Comment:
I always seen this one referred as "FAINT", as listed on https://en.wikipedia.org/wiki/ANSI_escape_code

Reply:
reverted the change